packages/unmock-cli/src/__tests__/commands/list.test.ts

```diff
@@ -0,0 +1,11 @@
+import { curlInternal } from "../../commands/curl";
+import { listInternal } from "../../commands/list";
+
+test("list works", async () => {
+  // this makes sure that some test has happened
+  await curlInternal("https://www.behance.net/v2/projects", {});
+  const log = jest.fn();
+  await listInternal({ log });
```

Comment:
Wouldn't it be better to have `log` as some store, so that we can check e.g. that `method`, `host`, `path` etc are all present in the output?